libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -16,24 +16,28 @@ import { NextDirective } from './directives/next.directive';
 
 import { HomeRoutingModule } from './home.routing';
 
+import { HomeImpactSectionComponent } from './components/home-impact-section/home-impact-section.component';
+import { ButtonsModule } from '@elewa-group/features/components/buttons';
+
+
 
 @NgModule({
   imports: [
     CommonModule,
     LayoutModule,
     NgOptimizedImage,
     CardsModule,
-    HomeRoutingModule
+    HomeRoutingModule,
+    ButtonsModule
   ],
   declarations: [
     HomePageComponent,
     HomeHeroSectionComponent,
     HomeJobsSectionComponent,
-    TeamAndPartnersComponent,
     ActivitySectionComponent,
-    NextDirective,
-    PrevDirective
```

Comment:
This directives are being used by the values carousel on mobile, should not be removed.

Reply:
resolved